code/modules/logging/log_holder.dm

```diff
@@ -1,5 +1,4 @@
-GLOBAL_DATUM_INIT(logger, /datum/log_holder, new)
-GLOBAL_PROTECT(logger)
+GLOBAL_REAL(Logger, /datum/log_holder)
```

Comment:
1. Why is this a GLOBAL_REAL?
2. It should've stayed `logger`

Reply:
Ugh the convenience here might be important enough, I want a global proc for this instead but I can't even think of a name that isn't `log` (and `Log` annoys me)

Reply:
> Due to issues related to logging to world at global creation Logger is a global real, which is setup in the same manner Config and GLOB is, by having Master do it in Initializations